### PR TITLE
Create initial readme

### PR DESCRIPTION
**Pull request by VasiliiGeneralov**
README.md file is created in this commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# EP
+
+Editor on patterns.
+
+# Building from source
+
+1. Get and build (vcpkg)[https://github.com/microsoft/vcpkg]
+1. The rest is done by `cmake` automatically:
+`cmake -S . -B ./build/ && cmake --build ./build`
```
Comment (VasiliiGeneralov) [inline README.md:9]: Make readable please

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Reference
+
+1. "Design Patterns: Elements of Reusable Object-Oriented Software",
+Erich Gamma, Richard Helm, Ralph Johnson, John Vlissides
```
Comment (VasiliiGeneralov) [inline README.md:14]: Any tips?

File: README.md (hunk 1 above)
Comment (VasiliiGeneralov) [reply README.md:9]: Please read from left to right

File: README.md (hunk 2 above)
Comment (VasiliiGeneralov) [reply README.md:14]: Pro tip: this is the book title

File: README.md (hunk 1 above)
Comment (goncharovnikita) [reply README.md:9]: 'cmake' occurs 3 times, consider refactor to conform DRY principle

Comment (VasiliiGeneralov) [reply README.md:9]: Added to backlog yes

**Review by goncharovnikita — APPROVED** [review]
Hm hm, nice, okay

**Pull request merged**